packages/common-scripts/index.d.ts

```diff
@@ -1,9 +1,22 @@
-import { TransactionSkeleton } from "@ckb-lumos/helpers"
-import { Cell, CellProvider } from "@ckb-lumos/base"
+import { TransactionSkeletonType } from "@ckb-lumos/helpers"
+import { Cell, CellProvider, Script, Header, PackedSince, HexString, Hash, PackedDao } from "@ckb-lumos/base"
 
 export type Address = string
 export type Config = any // TODO: define this type later
 
+export interface LocktimeCell {
+  cell: Cell
+  maximumCapacity: bigint
```

Comment:
Should it be `BigInt`? I'm not sure if case matters.

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-2.html#bigint